### PR TITLE
Fix import from external module

### PR DESCRIPTION
**Pull request by disrupted**

### CODE REVIEW

**Comment by irux** [comment]
Does this PR closes an opened issue ?

**Comment by disrupted** [comment]
> Does this PR closes an opened issue ?

no, it's a newly appeared issue that I thought of fixing right away

File: tests/cli/test_registry.py [hunk 1]
```diff
@@ -23,16 +24,35 @@ class Unrelated:
 
 
 def test_find_classes():
-    gen = _find_classes(MODULE, BaseModel)
+    gen = _find_classes(MODULE, PipelineComponent)
```
Comment (sujuka99) [inline tests/cli/test_registry.py:27]: what does `gen` mean here?

**Review by sujuka99 — APPROVED** [review]

File: tests/cli/test_registry.py (hunk 1 above)
Comment (disrupted) [reply tests/cli/test_registry.py:27]: generator

Comment (disrupted) [reply tests/cli/test_registry.py:27]: should I rename the variable? do you have a suggestion for a more clear name?

**Pull request merged**